src/ray/gcs/gcs_client/gcs_client.cc

```diff
@@ -139,5 +139,286 @@ std::pair<std::string, int> GcsClient::GetGcsServerAddress() const {
   return gcs_rpc_client_->GetAddress();
 }
 
+GcsSyncClient::GcsSyncClient(const GcsClientOptions &options) : options_(options) {}
+
+Status GcsSyncClient::Connect() {
+  grpc::ChannelArguments arguments = CreateDefaultChannelArguments();
+  arguments.SetInt(GRPC_ARG_MAX_RECONNECT_BACKOFF_MS,
```

Comment:
Maybe all these arguments should be a part of `CreateDefaultChannelArguments`? Otherwise, it will be prone to error and can easily have unsynced configuration between Python & cpp clients.

Reply:
I removed the `CreateDefaultChannelArguments` call now and just set exactly the same values that we set in the Python code to keep the changes due to this PR as small as possible!